### PR TITLE
fix(cmn): BaseServiceV2 throws on camelCase options

### PR DESCRIPTION
**Pull request by Inphi**
This fixes the `BaseServiceV2` ctor throwing with a `Error: missing required option: camelCase` error.

### CODE REVIEW

File: packages/common-ts/test/service-spec.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    })
+  }
+  protected async main() {
+    /* eslint-disable @typescript-eslint/no-empty-function */
+  }
+}
+
+describe('BaseServiceV2', () => {
+  it('base service ctor does not throw on camel case options', async () => {
+    // throws on error
```
Comment (roninjin10) [inline packages/common-ts/test/service-spec.ts:27]:
```suggestion
```

File: packages/common-ts/test/service-spec.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      name: 'test-service',
+      version: '0.0',
+      options,
+      optionsSpec: {
+        camelCase: { validator: validators.str, desc: 'test' },
+      },
+      metricsSpec: {},
+    })
+  }
+  protected async main() {
```
Comment (roninjin10) [inline packages/common-ts/test/service-spec.ts:20]: main being required is a bug imo

**Comment by roninjin10** [comment]
Might be worth doing a patch changeset

**Comment by Inphi** [comment]
will add a changeset in a bit

**Pull request merged**